src/examples/fake_device_demo.rb

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
+
+$LOAD_PATH.unshift(File.expand_path('../../lib', __FILE__))
+
+require "storage/fake_probing.rb"
+require "storage/fake_device_factory.rb"
+
+FILENAME = "fake-devicegraphs"
+
+fake_probing = Yast::Storage::FakeProbing.new
+devicegraph = fake_probing.devicegraph
+factory = Yast::Storage::FakeDeviceFactory.new(devicegraph)
+factory.load_yaml_file("#{FILENAME}.yml")
+
+fake_probing.to_probed
+puts("Disks:")
+probed = Yast::Storage::StorageManager.instance.probed
+fake_probing.dump_disks(probed)
```

Comment:
Also, it is technically not possible to create an anonymous device graph first and then start the FakeProbing mechanics: FakeProbing needs to be first to prevent libstorage-bgl from doing any probing when its first instance is created. That's basically the main thing FakeProbing does. Copying an arbitrary device graph to the "probed" device graph is just secondary functionality; the primary one is to set up the StorageManager / libstorage-bgl so that it doesn't do any real probing to begin with (which would require root permissions for one thing, and, more importantly, would pollute the "probed" device graph with the real devices from the host machine - which is not desired here).